bin/pycbc_multi_inspiral

```diff
@@ -93,6 +93,28 @@ def calculate_antenna_pattern(args, sky_pos_indices):
             )
     return antenna_pattern
 
+def slide_limiter(args):
+    '''
+    This function computes the number of shortslides used by the coherent
+    matched filter statistic to obtain as most background triggers as 
+    possible.
+
+    It it bounds the amount of slides to avoid counting triggers more 
+    than once.
+    '''
+    low, upp = 1, args.segment_length[args.instruments[0]]
+    if args.do_shortslides:
+        n_ifos = len(args.instruments)
+        stride_dur = args.segment_length[args.instruments[0]]/2
+        num_slides = np.int32(1 + n_ifos *\
```

Comment:
```suggestion
        num_slides = np.int32(1 + 
```

Reply:
This was necessary for storing time slides with the previous code, but is not needed here.